src/main/java/View/Web/Servlets/ConnectionServlet.java

```diff
@@ -0,0 +1,20 @@
+package View.Web.Servlets;
+
+import Control.ConfigClass;
+import Control.Connect.DbProvider;
+
+import javax.servlet.http.HttpServlet;
+
+public abstract class ConnectionServlet extends HttpServlet {
+
+    protected static final boolean trueIfRunningRemotely = ConfigClass.RUNNING_ON_SERVER;
+    protected DbProvider dbProvider;
```

Comment:
Teraz je instancia spojenia z databazou per servlet. Podla mna by mala byt len jedna instancia toho objektu pre vsetky routy spolu, ako to byva vo frameworkoch. Riesil by som to v nejakej onStart alebo onContextStart metode pre java web aplikaciu, ale zatial som s tym nerobil cize neviem presne ktora metoda je na to urcena.